docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -9,8 +9,8 @@ Jobs are the main unit of execution and monitoring in Dagster.
 
 A job does one of two things:
 
-- Materializes a selection of [Software-defined Assets](/concepts/assets/software-defined-assets)
-- Executes a [graph](/concepts/ops-jobs-graphs/graphs) of [ops](/concepts/ops-jobs-graphs/ops), which are not tied to software-defined assets
+- Materializes a selection of [Software-defined Assets](/concepts/assets/software-defined-assets). To learn more about materializing and monitoring assets, visit [Asset jobs](/concepts/assets/asset-jobs).
```

Comment:
Including this in here feels really weird to me, since this page is supposed to be about op jobs. In that case, why would the description include assets?